cat-gradle-sass-plugin/src/main/groovy/cc/catalysts/gradle/sass/SassExtension.groovy

```diff
@@ -0,0 +1,61 @@
+package cc.catalysts.gradle.sass
+
+import cc.catalysts.gradle.npm.AbstractNpmAwareExtension
+import cc.catalysts.gradle.npm.PackageJson
+import org.gradle.api.Project
+
+class SassExtension extends AbstractNpmAwareExtension {
+    File srcDir
+    String[] srcFiles
+    String cssPath
+    List<String> additionalArguments = [ ]
+    Map<String, String> npmDependencies = [
+        'node-sass'     : '4.5.3',
+    ]
+
+    Closure<String> cssFileName = {
+        return it.replaceAll('\\.s[ac]ss$', '.css')
+    }
```

Comment:
please remove the possibility to provide the css filenames -> they are not passed to the sass compiler so there is no need to allow configuration of them